skycoin-api/skycoin_crypto.c

```diff
@@ -345,6 +345,34 @@ int skycoin_address_from_pubkey(const uint8_t* pubkey, char* b58address, size_t*
     return 0;
 }
 
+int bitcoin_address_from_pubkey(const uint8_t* pubkey, char* b58address, size_t* size_b58address){
+    const curve_info* curve = get_curve_by_name(SECP256K1_NAME);
+
+    if (!pubkey_is_valid(curve->params, pubkey)) {
+        return 0;
+    }
+
+    uint8_t address[RIPEMD160_DIGEST_LENGTH + 1 + 4] = {0};
```

Comment:
Add similar as SKYCOIN_ADDRESS_CHECKSUM_LENGTH constant for Bitcoin checksum and use it instead of 4.